charts/management-portal/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "2.0.0"
 description: A Helm chart for RADAR-Base Management Portal to manage projects and participants throughout RADAR-base.
 name: management-portal
-version: 1.0.0
+version: 1.0.2
```

Comment:
why is it two version numbers?

Reply:
Had to fix something in [c4ad090](https://github.com/RADAR-base/radar-helm-charts/pull/96/commits/c4ad09028ab845da7f824b7bfad69b2f9946a4b0)